src/OneExplorer.ts

```diff
@@ -387,13 +388,12 @@ input_path=${filename}.${extname}
 }
 
 export class OneExplorer {
-  constructor(context: vscode.ExtensionContext, logger: Logger) {
-    const rootPath =
-        (vscode.workspace.workspaceFolders && (vscode.workspace.workspaceFolders.length > 0)) ?
-        vscode.workspace.workspaceFolders[0].uri :
-        undefined;
+  // TODO Support multi-workspace
```

Comment:
(Q) Why should we support multi-workspace?

Reply:
In usability's perspective. Why not? When a user have opened multiple workspaces and if only one of them is displayed inside ONE, the user may think that it's not much usable. Plus, there are several comments mentioning about multiple workspaces, so I thought some of us already thought of it.

Reply:
Do you mean [multi-root workspace](https://code.visualstudio.com/docs/editor/multi-root-workspaces)? I never heard about multi-workspace word in vscode. As I know, workspace in vscode is that are opened in a VS code window(instance). And vscode supports one or multiple root directories in one workspace.

Reply:
> Do you mean [multi-root workspace](https://code.visualstudio.com/docs/editor/multi-root-workspaces)?

Yes, I mean it :-D

Reply:
I edited the comment.